src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -61,7 +64,9 @@ public void navigate(final Project project) {
       final FlutterSdk sdk = FlutterSdk.getFlutterSdk(project);
       if (sdk != null) {
         try {
-          sdk.run(FlutterSdk.Command.DOCTOR, module, project.getBaseDir(), null);
+          final VirtualFile contentRoot = ModuleUtil.getProjectContentRoot(module);
+          assert contentRoot != null;
+          sdk.run(FlutterSdk.Command.DOCTOR, module, contentRoot, null);
```

Comment:
`flutter doctor` doesn't seem to need a project-specific base dir, does it? I suggest to get rid of 'harmful' `ModuleUtil` class at all! :)